src/v2/Components/Inquiry/Hooks/useInquiryContext.tsx

```diff
@@ -14,21 +21,23 @@ import { Spinner } from "@artsy/palette"
 export type Context = {
   askSpecialist: boolean
   collectorLevel?: number | null
+  isLoggedIn: boolean
   location?: Location | null
   phone?: string | null
   profession?: string | null
+  requiresReload: boolean
```

Comment:
This is the new addition here; we flip this flag after both login and sign up. And then the close function looks at it and decides to reload the page.